src/MgvLib.sol

```diff
@@ -53,30 +56,32 @@ library MgvLib {
    Some miscellaneous data types useful to `Mangrove` and external contracts */
   //+clear+
 
-  /* `SingleOrder` holds data about an order-offer match in a struct. Used by `marketOrder` (and some of its nested functions) to avoid stack too deep errors. */
+  /* `SingleOrder` holds data about an order-offer match in a struct. Used by `marketOrder` (and some of its nested functions) to avoid stack too deep errors. It is used in market order and cleaning. */
   struct SingleOrder {
     OLKey olKey;
     uint offerId;
+    /* The `offer` given to the maker will be cleaned of `prev`/`next` pointers. */
     Offer offer;
     /* `wants`/`gives` mutate over execution. Initially the `wants`/`gives` from the taker's pov, then actual `wants`/`gives` adjusted by offer's ratio and volume. */
```

Comment:
Is this true? 1) it is now called takerWants and not just wants (same for gives) and 2) the SingleOrder.takerWants/gives is the same as offer.gives/wants except for partial fill, so not from the takers pov.

Reply:
1) Good catch, should rename

2) It refers to mutation inside the code, which is still true given this in `MgvOfferTaking.execute`:

```solidity
        if ((mor.fillWants && offerGives <= fillVolume) || (!mor.fillWants && offerWants <= fillVolume)) {
          sor.takerWants = offerGives;
          sor.takerGives = offerWants;
        } else {
          if (mor.fillWants) {
            /* While a possible `offer.wants()=0` is the maker's responsibility, a small enough partial fill may round to 0, so we round up. It is immaterial but more fair to the maker. */
            sor.takerGives = sor.offer.tick().inboundFromOutboundUp(fillVolume);
            sor.takerWants = fillVolume;
          } else {
            sor.takerWants = sor.offer.tick().outboundFromInbound(fillVolume);
            sor.takerGives = fillVolume;
          }
```

Reply:
> Initially the `wants`/`gives` from the taker's pov
I would say its from the maker's pov and then only adjusted for partial fill in the end to:
>, then actual `wants`/`gives` adjusted by offer's ratio and volume.

Reply:
I'm confused, what do you mean "from taker's pov" or "from maker's pov"? Here I only refer to what mutates this value in memory.

Reply:
I'm referring to old line 61 / new line 65 - it mentions "from taker's pov"